dashboard/components/__init__.py

```
@@ -0,0 +1,5 @@
+from .metrics import MetricsDisplay
+from .visualizations import VulnerabilityChart, PerformanceGraph
+from .reports import ReportGenerator
+
+__all__ = ['MetricsDisplay', 'VulnerabilityChart', 'PerformanceGraph', 'ReportGenerator']
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)